lib/vnet/daemon/client_darwin.go

```diff
@@ -14,6 +14,9 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+//go:build vnetdaemon
+// +build vnetdaemon
```

Comment:
Because of `lib/vnet/daemon/common.go` which is imported by any platform, `lib/vnet/daemon` ends up being imported on macOS even when `vnetdaemon` is not used. So I added that build tag to all files in the daemon package except `common.go`.

This way the Obj-C code is not built if `vnetdaemon` is not provided as a build tag.